examples/README.md

```diff
@@ -1,24 +1,7 @@
 # Example user scripts and task lists
 
-This directory contains example scripts that use publicly available data (either BIDS or Cusack lab). Requirements of the specific analyses beyond those listed on the main project page are listed in the scripts. For an automated test, see aa_testcode/aatest. 
+This directory contains example scripts that use publicly available data (either BIDS or Cusack lab).
 
-#### demo_basic_aa.m & demo_basic_tasklist.xml
-Runs one session of one subject through a standard SPM-based fMRI analysis.
+The tutorials will help you get started with AA, and require only AA and SPM.
```

Comment:
The "official" abbreviation of Automatic Analysis is aa rather than AA to distinguish from other organisations, such as Alcoholics Anonymous and The Automobile Association.